### ISSUE TITLE
Web site colour scheme and matching styles

### ISSUE THREAD

**Issue by pmcclure852** [zhorzhz/park#14]
I've been thinking about the colour scheme for the site. When I used the colours from swatches, I couldn't get anything to look right with the logo. I think we should either change the logo or stick with the the grey it currently is or something quite similar (not as black as on the swatches), unless anyone can find something else they like.

Regarding the colours overall, I think the white background we've used on the home page with the grey header and footer is fine. I like the other colours on the home page too and I especially like the look of the text on the banner image at the top of the page.

I think it would be good if we could take that top banner format and reproduce it across the other pages.

Here are my suggestions:

- Attractions page - change background to white and change the banner image text to match the home page.

- Info page - while the tickets image is really good, I find it a little jarring when navigating to that page, the word is too large and doesn't quite match the other pages. I would suggest we choose another image and add text to match the other pages. The scaling of the corona virus information could be better -  is there a way to fix the width so the image is always the right size for the area and have the the details to the right wrap perhaps? 

I'm happy to help with any of this BTW :)

**Comment by KimberlyO367** [comment]
Yeah! I agree with all this, I think we will have more synergy on the page too if we do the above. Almost forgot! The only other thing to add is I think we should prob remove the JavaScript in the map on the attractions page

**Comment by pmcclure852** [comment]
I forgot the map - yes, I think we should revise the map - initially I'd imagined a map of the park that you could click to see details of the attractions but I think that's too much for where we're at right now. 

I think a nice little google maps thing could be ok with our imaginary address if we can use one without javascript (not possible?), or maybe we scrap the map idea and do something else.
